example/asm/shellcode.py

```diff
@@ -96,8 +96,13 @@
     patches = new_patches
 
 print patches
-for offset, raw in patches.items():
-    virt[offset] = raw
+if isinstance(virt, StrPatchwork):
```

Comment:
Do you plan to update `StrPatchwork` API too?
By adding a `.set()` method, in order to be independent of the sublayer (`StrPatchwork`, elfesteem `virt` wrapper, ...)